filebeat/beater/crawler.go

```diff
@@ -105,7 +105,7 @@ func (c *crawler) Start(
 		}()
 	}
 
-	log.Infof("Loading and starting Inputs completed. Enabled inputs: %v", len(c.inputs))
+	log.Infof("Loading and starting Inputs completed. Enabled inputs: %d", len(c.inputs))
 
```

Comment:
@AndersonQ The len() is a good start but let's be more precise like having a detailed count for each input. IE. `Filestream input: 2`, same comment for this log [statement](https://github.com/elastic/beats/pull/30534/files#diff-bf31b3628cee561576e75d284fc77cd6d5423460593d39f7309685b601c3070fR119).

Reply:
I'm logging the input config keys now on `startInput` now.
The `inputs` field doesn't have much information to log, it's an implementation of `Runner` :/